app/views/content/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -130,84 +151,90 @@ You can apply for a salaried teacher training course, when you’ll earn money w
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa">Find out more about applying for the right visa for your teacher training course</a>
 
-## 6. Plan your move to the UK
+### 6. Plan your move to the UK
 
-The [UK Council for International Student Affairs delivers independent advice](https://www.ukcisa.org.uk/Information--Advice) about all aspects of the international student experience, including immigration, finding a place to live and opening a bank account. Their [Student Advice Line](https://www.ukcisa.org.uk/About-UKCISA/Contact-us) also offers support over the phone.
+Visit the [UK Council for International Student Affairs](https://www.ukcisa.org.uk/Information--Advice) for advice about immigration, finding a place to live and opening a bank account. Their [Student Advice Line](https://www.ukcisa.org.uk/About-UKCISA/Contact-us) also offers support over the phone.
 
 Your teacher training provider may also be able to help you plan your move to the UK  – contact them directly to ask.
 
 
-## 7. Complete your teacher training and gain qualified teacher status (QTS)
+### 7. Complete your teacher training and gain qualified teacher status (QTS)
 
-If you’re accepted on to a teacher training course in England, you’ll train for one year (or longer if you’re part-time) to gain [QTS](https://www.gov.uk/guidance/qualified-teacher-status-qts).
+If you’re accepted on to a teacher training course in England, you’ll train for one year (or longer if you’re part-time) to gain [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts).
 
 You may need to apply for a different visa to work in England as a qualified teacher. Our [guidance for non-UK nationals wishing to teach in England](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk) explains the skilled worker visa and covers other ways you can update, extend or switch your visa when you finish your training.
 
-## 8. Apply for your first teaching job
+### 8. Apply for your first teaching job
 
 When you have qualified teacher status, you will be able to apply for paid teaching jobs in all schools in England.
 
-<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job" class="button">Learn more about your first teaching job in England</a>
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job">You can learn more about your first teaching job in England</a>
 
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job">Learn more about your first teaching job in England</a>
```

Comment:
Repeats above link. Assume this was to replace the above and the above should be deleted?